### PR TITLE
[WIP] Add run_exports for htslib

### PR DESCRIPTION
**Pull request by kyleabeauchamp**
:information_source:
Bioconda has finished the [GCC7 migration](https://github.com/bioconda/bioconda-recipes/issues/13578). If you are dealing with C/C++ or Python package it can be that you need to rebuild other dependent packages. [Bioconda utils - update-pinning](https://bioconda.github.io/updating.html#updating-recipes-for-a-pinning-change) will assist you with that. If you have any questions please use [issue 13578](https://github.com/bioconda/bioconda-recipes/issues/13578).

----------------

* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/htslib/meta.yaml [hunk 1]
```diff
@@ -6,6 +6,8 @@ package:
 
 build:
   number: 5
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
```
Comment (bgruening) [inline recipes/htslib/meta.yaml]: @kyleabeauchamp the syntax looks like that and it needs to be part of /build/run_exports/.

Do you know which versions are API stable? x.x. or x.x.x or x?

Comment (kyleabeauchamp) [reply recipes/htslib/meta.yaml]: I'm not sure how well they abide by versioning, but I would guess x.x, i.e., I would trust anything in the 1.9.X series for this build...

**Comment by kyleabeauchamp** [comment]
PS: we should probably wait until https://github.com/bioconda/bioconda-utils/issues/379 is resolved before merging this.

**Comment by bgruening** [comment]
@kyleabeauchamp can you keep track of this and merge if you think its ok. Thanks!

**Pull request merged**